CONTRIBUTING.md

```diff
@@ -56,10 +56,26 @@ eval "$(pyenv init -)"
 eval "$(pyenv virtualenv-init -)"
 ```
 
-For Linux (or on the Mac, if you don't want to use `brew`) you can use
-[pyenv/pyenv-installer](https://github.com/pyenv/pyenv-installer) to
-install the necessary tools.  When you are finished you will need to
-add the same two lines above to your profile.
+For Linux, Windows Subsystem for Linux (WSL), or on the Mac (if you don't want
+to use `brew`) you can use [pyenv/pyenv-installer](https://github.com/pyenv/pyenv-installer)
+to install the necessary tools. Before running this ensure that you have installed
+the prerequisites for your platform according to the [`pyenv` wiki page](https://github.com/pyenv/pyenv/wiki/common-build-problems)
+.
+
+On WSL you should treat your platform as whatever Linux distribution you've
+chosen to install.
+
+Once you have installed `pyenv` you will need to add the following lines to your
+`.bashrc`:
```

Comment:
The installer script outputs instructions for what to add at the end if the command isn't seen [here](https://github.com/pyenv/pyenv-installer/blob/master/bin/pyenv-installer#L89). So maybe instead just note to follow the installer's directions regarding what to add?

Reply:
We can do both- since most people will have bash, what you wrote originally is good.  If they don't run bash, we can direct them to follow the instructions output by the installer- maybe something like so:

> Once you have installed `pyenv` you will need to add the following lines to your
> `.bashrc`:
> ...
> If you are running a shell other than `bash`, follow the instructions output by the `pyenv` installer.

Reply:
Sounds good, I added a line to that effect in 08e7502.